### ISSUE TITLE
Comment app

### ISSUE THREAD

**Issue by mycarta** [mycarta/t21-hack-footprint#10]
Assigned everyone to this collaborative effort. 
Let's go back to the app and heavily comment it. I'd like to see it all the way to ELI5 level (explain it like I am 5). Egoistically for my benefit, but also anyone who wants to dig deeper into the code.

**Comment by WesleyTheGeolien** [comment]
Depending on the scale / lifeline of this project would it be worth setting up a GitHub Pages for documentation?

**Comment by mycarta** [comment]
> Depending on the scale / lifeline of this project would it be worth setting up a GitHub Pages for documentation?

It may be worth the while. Let's get a few more easy enhancements in, then we can have that discussion as a team...
